python/src/opendp/context.py

```diff
@@ -364,6 +367,9 @@ def _is_distance(p, v):
     if sum(1 for p, v in locals().items() if _is_distance(p, v)) != 1:
         raise ValueError("Must specify exactly one distance.")
 
+    if local:
```

Comment:
Should there be any more validation here? I would guess some combinations don't make sense, but I could be wrong.